doc/cephfs/cephfs-shell.rst

```diff
@@ -14,6 +14,17 @@ Options :
     -b, --batch FILE      Process a batch file.
     -t, --test FILE       Test against transcript(s) in FILE
 
+
+.. note::
+
+    For running cephfs-shell, at least one metadata pool and one data pool should be active.
+    If CephFS installed through source, execute cephfs-shell in the build directory.
+    It can also be executed as following:
+
+.. code:: bash
+
+    $ ./../src/tools/cephfs/cephfs-shell
+
```

Comment:
I would also add a note that `cephfs-shell` requires a recent version of the Cmd2 module. This can be done in a virtualenv:

````
$ (virtualenv -p python3 venv && source venv/bin/activate && pip3 install cmd2)
````

Then:

$ (source vstart_environment.sh && source venv/bin/activate && python3 ../src/tools/cephfs/cephfs-shell)

Reply:
Updated the patch, according to the comments.